GameServerLib/Chatbox/Commands/MobsCommand.cs

```diff
@@ -45,7 +45,7 @@ public override void Execute(Peer peer, bool hasReceivedArguments, string argume
             var client = _playerManager.GetPeerInfo(peer);
             foreach (var unit in units)
             {
-                 _game.PacketNotifier.NotifyPing(client, unit.Value.X, unit.Value.Y, 0, Pings.PING_DANGER);
+                 _game.PacketNotifier.NotifyPing(client, unit.Value.Position.X, unit.Value.Position.Y, 0, Pings.PING_DANGER);
```

Comment:
Maybe change `NotifyPing` to also take a `Vector2`?

Reply:
Yeah i left that as is, because im not so familiar with the networking functionality, also if i remember right Networking is currently worked on and will change.